cognite_toolkit/cdf_tk/load.py

```diff
@@ -855,12 +861,50 @@ def create(self, items: Sequence[T_Resource], drop: bool, filepath: Path) -> T_R
                         if item.external_id == ext_id:
                             items.remove(item)
                 try:
-                    return ExtractionPipelineList(self.client.extraction_pipelines.create(items))
+                    extraction_pipelines = self.client.extraction_pipelines.create(items)
                 except Exception as e:
                     print(f"[bold red]ERROR:[/] Failed to create extraction pipelines.\n{e}")
                     self.ToolGlobals.failed = True
-                    return None
-            return None
+                    return ExtractionPipelineList([])
+
```

Comment:
You need this one as well to handle the case when all items are duplicated. 
```suggestion
                    return ExtractionPipelineList([])
            else:
                print(
                    f"  [bold yellow]WARNING:[/] {len(e.duplicated)} out of {len(items)} extraction pipelines already exist(s)."
                )
                return ExtractionPipelineList([])
```